samples/sample-04/build.gradle

```diff
@@ -0,0 +1,87 @@
+buildscript {
+    ext {
+        kotlinVersion = "1.3.10"
+        springBootVersion = "2.1.1.RELEASE"
+    }
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}")
+        classpath("org.jetbrains.kotlin:kotlin-allopen:${kotlinVersion}")
+        classpath("org.jetbrains.kotlin:kotlin-noarg:${kotlinVersion}")
+        classpath("io.spring.gradle:dependency-management-plugin:v1.0.6.RELEASE")
+    }
+}
+
+apply plugin: "kotlin"
+apply plugin: "kotlin-spring"
+apply plugin: "org.springframework.boot"
+apply plugin: "io.spring.dependency-management"
+apply plugin: "kotlin-allopen"
+apply plugin: "kotlin-noarg"
+apply plugin: "kotlin-jpa"
+
+
+group "com.example"
+version "0.0.1-SNAPSHOT"
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven { url 'http://repo.spring.io/milestone' }
+}
+noArg{
+    annotation("com.remusrd.notesample.domain.annotation.NoArg")
+}
+allOpen{
+    annotation("com.remusrd.notesample.domain.annotation.Open")
+}
+
+ext{
+    arrow_version = "0.8.1"
+}
+
+dependencies {
+    // Kotlin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib"
+    implementation "org.jetbrains.kotlin:kotlin-reflect"
+    implementation "com.fasterxml.jackson.module:jackson-module-kotlin"
+    implementation "io.arrow-kt:arrow-core:$arrow_version"
+    implementation "io.arrow-kt:arrow-data:$arrow_version"
+
+    // Spring Boot
+    implementation "org.springframework.boot:spring-boot-starter-web"
+    implementation "org.springframework.boot:spring-boot-starter-data-jpa"
+    implementation "org.springframework.kafka:spring-kafka"
+    implementation "com.fasterxml.jackson.module:jackson-module-kotlin:2.9.7"
```

Comment:
We don't need an explicit version for this dependency as well. It is managed by the `jackson-bom`: https://github.com/FasterXML/jackson-bom/blob/master/pom.xml#L258